Source/WebCore/Modules/webaudio/AudioWorkletProcessor.cpp

```diff
@@ -80,12 +80,12 @@ static JSObject* toJSObject(JSValueInWrappedObject& wrapper)
     return wrapper ? jsDynamicCast<JSObject*>(wrapper.getValue()) : nullptr;
 }
 
-static JSFloat32Array* constructJSFloat32Array(JSGlobalObject& globalObject, unsigned length, const float* data = nullptr)
+static JSFloat32Array* constructJSFloat32Array(JSGlobalObject& globalObject, unsigned length, std::span<const float> data = { })
 {
     constexpr bool isResizableOrGrowableShared = false;
     auto* jsArray = JSFloat32Array::create(&globalObject, globalObject.typedArrayStructure(TypeFloat32, isResizableOrGrowableShared), length);
-    if (data)
-        memcpy(jsArray->typedVector(), data, sizeof(float) * length);
+    if (!data.empty())
```

Comment:
Is this an important optimization?

Reply:
I don't think so, I was just matching the original code. I'll drop the check.